ui/apos/components/AposExportModal.vue

```diff
@@ -28,13 +28,10 @@
 
             <div class="apos-export__settings-row">
               <div>{{ $t('aposImportExport:exportModalDocumentFormat') }}</div>
-              <AposContextMenu
-                disabled
-                :button="{
-                  label: 'ZIP',
-                  icon: 'chevron-down-icon',
-                  modifiers: ['icon-right', 'disabled']
-                }"
+              <AposSelect
+                :choices="extensions"
+                :selected="extension"
+                @change="onExtensionChange"
```

Comment:
yeah, we said only ZIP...
but since gzip was added for the sake of testing another library, let's keep these 2 formats